app/views/schools/participants/index.html.erb

```diff
@@ -5,50 +5,91 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <span class="govuk-caption-xl"><%= @school.name %></span>
-    <h1 class="govuk-heading-xl">Your ECTs and mentors</h1>
+    <h1 class="govuk-heading-xl">Manage mentors and ECTs</h1>
 
-    <p class="govuk-body">We’ll check your early career teacher (ECTs) and mentors details with the Teaching Regulation Agency to make sure they’re eligible for funding for the programme.</p>
-    <p class="govuk-body">In the meantime ECTs can begin training with their training provider.</p>
+    <div class="wrapper-dashboard-v2 manage-mentors-ects">
+      <%= govuk_button_link_to(
+            "Add ECT or mentor",
+            who_schools_add_participants_path(cohort_id: @school.school_cohorts.dashboard_cohorts.first.start_year),
+            class: "govuk-link--no-visited-state",
+          ) %>
 
-    <div class="govuk-inset-text">
-      <ul class="govuk-list">
-        <li><%= govuk_link_to "Add an ECT or mentor", who_schools_add_participants_path(cohort_id: @school_cohort.cohort.start_year), class: "govuk-link--no-visited-state" %></li>
-        <% unless current_user.mentor? %>
-          <li><%= govuk_link_to "Add yourself as a mentor", add_schools_participants_path(cohort_id: @school_cohort.cohort.start_year, type: :self), class: "govuk-link--no-visited-state" %></li>
-        <% end %>
-      </ul>
-    </div>
-  </div>
-  <div class="govuk-grid-column-full">
-    <% if @categories.any? %>
-      <%= govuk_tabs(title: 'Contents') do |component| %>
-        <% if participants_active?(@categories.ect_categories) %>
-          <% component.tab(label: 'ECTs') do %>
-            <%= render partial: 'ects', locals: { ect_profiles: @categories.ect_categories } %>
-          <% end %>
+      <% if actions?(@participants) %>
+        <% if @participants.orphan_ects.any? %>
+          <%= govuk_warning_text(text: "You need to assign a mentor to these ECTs.") %>
+          <ul class="govuk-list">
+            <% @participants.orphan_ects.sort_by(&:participant_full_name).each do |ect| %>
+              <li>
+                <%= govuk_link_to(ect.participant_full_name, schools_participant_path(id: ect.participant_profile_id,
+                                                                                      school_id: @school),
+                                  no_visited_state: true,
+                                  classes: ["govuk-\!-padding-bottom-static-1 govuk-\!-margin-bottom-static-0"]) %>
+              </li>
+            <% end %>
+          </ul>
+          <br>
         <% end %>
+      <% end %>
+
+      <% @participants.mentors.keys.sort_by(&:participant_full_name).each do |mentor| %>
+        <div class="group-card">
+          <%= govuk_summary_list(actions: false, borders: nil) do |list| %>
+            <% list.row do |row| %>
+              <% row.key(text: "Mentor") %>
+            <% end %>
```

Comment:
I don't think this works from a markup point of view. It will output a `<dt>Mentor</dt>` and multiple can be used, but, from [the docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dt):

> It is usually followed by a \<dd\> element; however, multiple \<dt\> elements in a row indicate several terms that are all defined by the immediate next \<dd\> element.

This means if we have more than one mentor the 'heading' will only apply to the first.

I think this should be a heading, which will give the page some hierarchy:

```suggestion
          <h2 class="govuk-heading-m">Mentor</h2>
          <%= govuk_summary_list(actions: false, borders: nil) do |list| %>
```

To a screen reader, when there's a big list of mentors, the page hierarchy will sound like:

> Manage mentors and ECTs, Mentor, Mentor, Mentor, Mentor

We probably want some visually hidden text in there too so it's a bit more informative.

```suggestion
          <h2 class="govuk-heading-m">Mentor <span class="govuk-visually-hidden>- <%= mentor.full_name %></span></h2>
          <%= govuk_summary_list(actions: false, borders: nil) do |list| %>
```

Reply:
Done